.github/ISSUE_TEMPLATE/1-bugreport.yml

```diff
@@ -0,0 +1,55 @@
+name: "Bug Report"
+description: "File a bug report"
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report!
+  - type: input
+    id: version
+    attributes:
+      label: "Version"
+      description: "Which node version are you running?"
+      placeholder: "v0.1.0"
+    validations:
+      required: true
+  - type: textarea
+    id: other-version
+    attributes:
+      label: "Other packages versions"
+      description: "Let us know of the versions of any other packages used. For example, which version of the client are you using?"
+      placeholder: "miden-client: v0.1.0"
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: "What happened?"
+      description: "Describe the behaviour you are experiencing."
+      placeholder: "Tell us what you see!"
+    validations:
+      required: true
+  - type: textarea
+    id: expected-to-happen
+    attributes:
+      label: "What should have happened?"
+      description: "Describe the behaviour you expected to see."
+      placeholder: "Tell us what should have happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce-steps
+    attributes:
+      label: "How can this be reproduced?"
+      description: "If possible, describe how to replicate the unexpected behaviour that you see."
+      placeholder: "Steps!"
+    validations:
+      required: true
```

Comment:
I wonder if this is a bit too rigid. Maybe a single text area with a hint "Tell use what happened, what should have happened, and how it can be reproduced." would be sufficient?

The main argument for this is to reduce friction when filing a bug. The counter-argument is maybe that we do want to create friction so that only people who are willing to take extra time file bugs.

Reply:
I would prefer to leave this as is. Not every person reporting a bug is a developer. Sometimes they are just unaware how to make a proper bug report and what even a MRE is.

Reply:
I will do the changes, but having a MRE is essential to any bug report, and a bug report without it is incomplete and most of the times useless.

Reply:
Generally agree - but in some cases this may not be easy to provide (e.g., [this](https://github.com/0xPolygonMiden/miden-base/issues/443)). I would rather capture such bugs without MRE and then we can figure out how to reproduce via comments.

Reply:
Your call, but one of the first comments in that issue is asking for a MRE https://github.com/0xPolygonMiden/miden-base/issues/443#issuecomment-1913532741